pino.js

```diff
@@ -250,6 +250,19 @@ Object.defineProperty(pinoPrototype, 'addLevel', {
   value: addLevel
 })
 
+function isLevelEnabled (logLevel) {
+  var logLevelVal = this.levels.values[logLevel]
+  if (logLevelVal && logLevelVal >= this._levelVal) {
```

Comment:
How about `return logLevelVal && (logLevelVal >= this._levelVal)` instead of the `if` statement?

Reply:
Good suggestion, fixed.